src/index.html

```diff
@@ -22,6 +22,377 @@
+            alt="Company Name Logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container-sections">
+        <section
+          class="container"
+          data-qa="card"
+        >
+          <div class="container__img">
+            <img
+              src="images/imac.jpeg"
+              alt="IMac image"
+              class="container__img-imac"
+            />
+          </div>
+
+          <div class="container__texts-inform">
+            <h1 class="container__title">
+              APPLE A1419 iMac 27" Retina
+              <br />
+              5K Monoblock (MNED2UA/A)
+            </h1>
+            <div class="container__code">Product code: 195434</div>
+            <div class="container__stars-reviews">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="container__review-text">Reviews: 5</div>
+            </div>
+            <div class="container__price-container">
+              <div class="container__price-container-label">Price:</div>
+              <div class="container__price-container-price">$2,199</div>
+            </div>
+          </div>
```

Comment:
don't forget about indents.  Code style is very important
```suggestion
            </h1>
            
            <div class="container__code">Product code: 195434</div>
            
            <div class="container__stars-reviews">
              <div class="stars stars--4">
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
              </div>
              
              <div class="container__review-text">Reviews: 5</div>
            </div>
            
            <div class="container__price-container">
              <div class="container__price-container-label">Price:</div>
              <div class="container__price-container-price">$2,199</div>
            </div>
          </div>
```